variables.tf

```diff
@@ -332,3 +332,12 @@ variable "cluster_encryption_config" {
   }))
   default = []
 }
+
+variable "eks_fargate_profiles" {
+  description = "EKS Fargate profiles"
+  type = map(object({
+    namespace = string
+    labels    = map(string)
```

Comment:
would we need different tags by profile ? If yes, we can probably add a `additional_tags` like what we already have in node group submodule https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/node_groups/node_groups.tf#L40-L43

Reply:
I don't see why that would be necessary.

Reply:
I don't use fargate, but you mean that no one would want to do something like 
```hcl
eks_fargate_profiles = {
  profile-1 = {
    namespace = "namespace-1"
    labels = {
      app = "foo"
    },
    additional_tags = {
      ownner = "team-foo"
    }
  },
  profile-2 = {
    namespace = "namespace-2"
    labels = {
      app = "bar"
    },
    additional_tags = {
      ownner = "team-bar"
    }
  }
}

}
```